views/tutorInfo.html

```diff
@@ -14,12 +14,23 @@ <h2 class="mt-3 text-center">{{ @tutor["tutor_first"] }} {{ @tutor["tutor_last"]
             <p class="mt-3"><span class="font-weight-bold mr-1">SSN:</span> {{ @ssn }}</p>
             <p class="mt-3"><span class="font-weight-bold mr-1">Bio:</span> {{ @tutor["tutor_bio"] }}</p>
             <form action="#" method="get" enctype="multipart/form-data">
-                <button type="submit" class="btn btn-info download" name="download" value="{{ @tutor['user_id'] }}">Downloads
+                <button type="submit" class="btn btn-info download" name="download" value="{{ @tutor['user_id'] }}">
+                    Download Image
                 </button>
+                <div class="mt-4">
```

Comment:
I would maybe add a line comment about what section of the page this div is.

Reply:
I will for sure! thank you